templates/directives/sender.html

```diff
@@ -1,13 +1,14 @@
-<div class="sender" ng-if="message">
+<div ng-class="{sender: true, 'small-font': sentBy}" ng-if="message">
```

Comment:
I think you can just have...

```html
<div class="sender" ng-class="{'small-font': sentBy}" ng-if="message">
```

I think that's a bit easier to read.

Reply:
Done.